### PR TITLE
[WIP] Changes to Prompts Landing page and sidebar.

### PR DESCRIPTION
**Pull request by katjacksonWB**
## Description

Adds more content to the "Prompts Platform" page to give users more context about W&B's LLM offerings. ~~Also, changes the page category name to "LLMs" and changes the information architecture of the page for better flow.~~
## Ticket

[DOCS-772](https://wandb.atlassian.net/browse/DOCS-772)

## Checklist

Check if your PR fulfills the following requirements. Put an `X` in the boxes that apply.

- [x] Files I edited were previewed on my local development server with `yarn start`. My changes did not break the local preview.
- [x] Build (`yarn docusaurus build`) was run locally and successfully without errors or warnings.
- [x] I merged the latest changes from `main` into my feature branch before submitting this PR.

[DOCS-772]: https://wandb.atlassian.net/browse/DOCS-772?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/guides/prompts_platform.md [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
 
-![](@site/static/images/general/prompts.png)
\ No newline at end of file
+## Monitor
+
+[LLM Monitoring](/guides/weave/prod-mon) provides large language model usage monitoring and diagnostics. Start simply, then customize and evolve your monitoring analytics over time.
+
+- Organizations can track and manage prompts usage statistics like cost across teams
+- Individuals can filter and drill down to analyze prompt performance, observe all inputs sent to the prompt and track custom metrics to use for analysis
+- (Coming soon) Use this monitored data to improve your pipeline by providing feedback and fine-tuning or evaluating with it.
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Edit this line to make it timeless. https://developers.google.com/style/timeless-documentation?hl=en

File: docs/guides/prompts_platform.md [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
+- Monitor and observe LLMs to better understand and evaluate performance, usage, and budgets.
 
-![](@site/static/images/general/prompts.png)
\ No newline at end of file
+## Monitor
+
+[LLM Monitoring](/guides/weave/prod-mon) provides large language model usage monitoring and diagnostics. Start simply, then customize and evolve your monitoring analytics over time.
+
+- Organizations can track and manage prompts usage statistics like cost across teams
+- Individuals can filter and drill down to analyze prompt performance, observe all inputs sent to the prompt and track custom metrics to use for analysis
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Replace "Individuals" with "you".  https://developers.google.com/style/person?hl=en

**Review by ngrayluna — CHANGES_REQUESTED** [review]
See comments

File: docs/guides/prompts_platform.md [hunk 3]
```diff
@@ -1,15 +1,37 @@
 ---
 displayed_sidebar: default
 ---
+# LLMs
 
-# W&B Prompts
-Gain a deeper understanding of your LLMs pipelines and prompts performance. Drill into your architectures for easy debugging, configure custom visualizations for the specific analysis you need, and maintain full observability over LLM performance, usage, and budget.
+Evaluating the performance of Large Language Models (LLMs) can be difficult. Weights and Biases offers several tools to streamline the evaluation process, providing a visual way to analyze your generative models.
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Replace all instances of "Weights & Biases" with "W&B"

Comment (ngrayluna) [reply docs/guides/prompts_platform.md]: This sentence 

"Weights and Biases offers several tools to streamline the evaluation process, providing a visual way to analyze your generative models."

is a little confusing. Suggest breaking it up into two or simplifying it? E.g. 

"Use W&B [BLANK and BLANK] to evaluate Large Language Models (LLMS) and analyze generative models".

or 

"Use W&B Prompts to visualize...". Monitor your large language models with LLM Monitoring.

File: docs/guides/prompts_platform.md [hunk 4]
```diff
@@ -1,15 +1,37 @@
...
 
+## Visualize
 
-W&B Prompts is the solution for building and monitoring LLM-based apps. Software developers, prompt engineers, ML practitioners, data scientists, and other stakeholders working with LLMs need cutting-edge tools to:
+[W&B Prompts](/guides/prompts) is a suite of LLMOps tools built for the development of LLM-powered applications. Use W&B Prompts to visualize and inspect the execution flow of your LLMs, analyze the inputs and outputs of your LLMs, view the intermediate results and securely store and manage your prompts and LLM chain configurations.
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Missing complete markdown file path

File: docs/guides/prompts_platform.md [hunk 5]
```diff
@@ -1,15 +1,37 @@
...
+W&B Prompts provides several solutions for building and monitoring LLM-based apps. Software developers, prompt engineers, ML practitioners, data scientists, and other stakeholders working with LLMs need cutting-edge tools to:
 
+- Explore and debug LLM chains and prompts with greater granularity.
+- Monitor and observe LLMs to better understand and evaluate performance, usage, and budgets.
 
-![](@site/static/images/general/prompts.png)
\ No newline at end of file
+## Monitor
+
+[LLM Monitoring](/guides/weave/prod-mon) provides large language model usage monitoring and diagnostics. Start simply, then customize and evolve your monitoring analytics over time.
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Missing full markdown path

File: docs/guides/prompts_platform.md [hunk 6]
```diff
@@ -1,15 +1,37 @@
...
+
+- Organizations can track and manage prompts usage statistics like cost across teams
+- You can filter and drill down to analyze prompt performance, observe all inputs sent to the prompt and track custom metrics to use for analysis
+- Usee monitored data to improve your pipeline by providing feedback and fine-tuning or evaluating with it.
+
+## Integrations
+
+Weights and Biases also has lightweight integrations for:
+
+- [LangChain](https://docs.wandb.ai/guides/integrations/langchain)
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Ah, to reference other docs you'll want to do relative markdown linking:  e.g. [Langchain](./integrations/langchain.md)

Do the same for the others.

File: docs/guides/prompts_platform.md [hunk 7]
```diff
@@ -1,15 +1,37 @@
...
+- [OpenAI API](https://docs.wandb.ai/guides/integrations/openai-api)
+- [OpenAI GPT-3.5 Fine-Tuning](https://docs.wandb.ai/guides/integrations/openai)
+- [Hugging Face Transformers](https://docs.wandb.ai/guides/integrations/huggingface)
+
+## Next Steps
+
+- Check out more detailed documentation on [Prompts](/guides/prompts) or [LLM Monitoring](/guides/weave/prod-mon).
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Add full path.

**Review by ngrayluna — CHANGES_REQUESTED** [review]
Requested changes.

**Comment by katjacksonWB** [comment]
@ngrayluna Does this look good to merge?

File: docs/guides/prompts_platform.md [hunk 8]
```diff
@@ -1,15 +1,37 @@
...
+# LLMs
 
-# W&B Prompts
-Gain a deeper understanding of your LLMs pipelines and prompts performance. Drill into your architectures for easy debugging, configure custom visualizations for the specific analysis you need, and maintain full observability over LLM performance, usage, and budget.
+Evaluating the performance of Large Language Models (LLMs) can be difficult. Use W&B Prompts and LLM Monitoring to streamline the evaluation process, providing a visual way to analyze your generative models.
 
+## Visualize
 
-W&B Prompts is the solution for building and monitoring LLM-based apps. Software developers, prompt engineers, ML practitioners, data scientists, and other stakeholders working with LLMs need cutting-edge tools to:
+[W&B Prompts](prompts/intro.md) is a suite of LLMOps tools built for the development of LLM-powered applications. Use W&B Prompts to visualize and inspect the execution flow of your LLMs, analyze the inputs and outputs of your LLMs, view the intermediate results and securely store and manage your prompts and LLM chain configurations.
```
Comment (ngrayluna) [inline docs/guides/prompts_platform.md]: Add "./" to paths. E.g. (./prompts/intro.md) to make it explicit.

**Review by ngrayluna — CHANGES_REQUESTED** [review]
Fix paths

**Comment by ngrayluna** [comment]
@katjacksonWB Let me know if you have any questions. Would be great to ship this out this week :)

**Pull request merged**